Bizyhood/Core.php

```diff
@@ -1945,6 +1959,73 @@ function bizyhood_plugin_action_links( $links ) {
        return $links;
     }
     
+    
+    function load_rss() {
```

Comment:
Okay Spyros. I tried out the code. There is an issue in error handling. If there are NO events in the feed or something is otherwise wrong, the code expects there to be an error.php underneath the rss views. It doesn't exist and throws a 500 error.

Could you please update the error handling?